src/main/java/ca/bc/gov/backendstartapi/config/SecurityConfig.java

```diff
@@ -64,27 +63,28 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 
   private Converter<Jwt, AbstractAuthenticationToken> converter() {
     JwtAuthenticationConverter converter = new JwtAuthenticationConverter();
-    converter.setJwtGrantedAuthoritiesConverter(roleConverter());
+    converter.setJwtGrantedAuthoritiesConverter(ROLE_CONVERTER);
     return converter;
   }
 
-  private Converter<Jwt, Collection<GrantedAuthority>> roleConverter() {
-    return jwt -> {
-      final JSONArray realmAccess = (JSONArray) jwt.getClaims().get("client_roles");
-      List<GrantedAuthority> authorities = new ArrayList<>();
-      if (ObjectUtil.isEmptyOrNull(realmAccess)) {
-        String sub = String.valueOf(jwt.getClaims().get("sub"));
-        if (sub.startsWith("service-account-nr-fsa")) {
-          authorities.add(new SimpleGrantedAuthority("ROLE_user_read"));
-          authorities.add(new SimpleGrantedAuthority("ROLE_user_write"));
+  /**
```

Comment:
Any reason why it was changed to static?

Reply:
Nothing special. Just the fact that it's independent of instance members.

Reply:
But now I believe that it isn't to be used here: from what I gather, this class will be instantiated and its methods will be executed only once, and making the attribute static will keep something unused in memory (it's not much, but it defeats the principle).